libmisc/xgetXXbyYY.c

```diff
@@ -69,15 +69,19 @@
 			/* Build a result structure that can be freed by
 			 * the shadow *_free functions. */
 			LOOKUP_TYPE *ret_result = DUP_FUNCTION(result);
+			if (NULL == result) {
+				fprintf (log_get_logfd(),
+				         _("%s: out of memory\n"),
+				         "x" STRINGIZE(FUNCTION_NAME));
+				exit (13);
+			}
```

Comment:
This looks like a typo in the NULL check.  You probably meant `ret_result == NULL`, right?